metron-streaming/Metron-DataLoads/pom.xml

```diff
@@ -167,56 +225,14 @@
         </resources>
         <plugins>
             <plugin>
-                <!-- Separates the unit tests from the integration tests. -->
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.12.4</version>
                 <configuration>
-                    <!-- Skip the default running of this plug-in (or everything is run twice...see below) -->
-                    <argLine>-Xmx2048m -XX:MaxPermSize=256m -Djava.util.logging.config.file=logging.properties</argLine>
+                    <argLine>-Xmx2048m -XX:MaxPermSize=256m -XX:-UseSplitVerifier</argLine>
                     <skip>true</skip>
-                    <!-- Show 100% of the lines from the stack trace (doesn't work) -->
                     <trimStackTrace>false</trimStackTrace>
-
                 </configuration>
-                <executions>
```

Comment:
So, I don't mind this, but this change will make the "integration tests" run as part of the unit test cycle and people have gotten touchy in the past due to the length that it adds to mvn package.  I'm generally for it as it ensures we get the full suite, but I wanted to make sure it was intended and we're ok with it in general.